api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/IgnoredParameters.scala

```diff
@@ -22,7 +22,8 @@ object IgnoredParameters {
     "r2_stopping", // All
     "max_confusion_matrix_size", // Deep Learning
     "col_major", // Deep Learning
-    "max_hit_ratio_k") // GBM, DRF, Deep Learning
+    "max_hit_ratio_k", // GBM, DRF, Deep Learning
+    "loading_name") // GLRM
```

Comment:
Thanks @wendycwong for the comment! I think this is already addressed. We generate parameters from h2o V3 classes. [This list](https://github.com/h2oai/sparkling-water/pull/2224/files#diff-47e4179c219099038269faa68f455740R21) indicates parameters that are deprecated in H2O-3 and won't be generated to SW API. Since `representation_name` is defined on V3 class and it's not on a blacklist anywhere, the relevant parameter will be generated. [Here](https://github.com/h2oai/sparkling-water/pull/2224/files#diff-51830a9432a74304b2536d848462099dR132) is a test using `representation_name`.